sekoia_automation/asset_connector/models/ocsf/device.py

```diff
@@ -109,6 +129,41 @@ class Device(BaseModel):
     location: GeoLocation | None = None
     os: OperatingSystem | None = None
     hostname: str
+    autoscale_uid: str | None = None
+    boot_time: str | None = None
+    boot_uid: str | None = None
+    created_time: str | None = None
+    description: str | None = None
+    domain: str | None = None
+    eid: str | None = None
+    first_seen_time: str | None = None
+    groups: list[Group] | None = None
+    hypervisor: str | None = None
+    iccid: str | None = None
+    imeis: list[str] | None = None
+    ip: str | None = None
+    is_backed_up: bool | None = None
+    is_compliant: bool | None = None
+    is_managed: bool | None = None
+    is_mobile_account_active: bool | None = None
+    is_personal: bool | None = None
+    is_shared: bool | None = None
+    is_supervised: bool | None = None
+    is_trusted: bool | None = None
+    last_seen_time: str | None = None
```

Comment:
According to [this documentation](https://schema.ocsf.io/1.5.0/objects/device) the type of the `last_seen_time` field is `Timestamp` that is a long according to [this doc](https://schema.ocsf.io/1.5.0/data_types)